src/scroll-timeline-base.js

```diff
@@ -515,6 +309,11 @@ function getScrollParent(node) {
   if (!node)
     return undefined;
 
+  if (!(node instanceof HTMLElement)) {
+     return node.parentNode ? getScrollParent(node.parentNode)
```

Comment:
So, walking the parentNode chain is incorrect. We should be walking the containing block chain. E.g. a fixed position element's source should be `document.scrollingElement` in the absence of any fixed position containing ancestors, and similarly absolute position often escapes scrollers. This is a pre-existing issue but how about adding a TODO for this?

Reply:
Added a TODO.